panel/reactive.py

```diff
@@ -1221,9 +1221,27 @@ def _convert_column(
         dtype = old_values.dtype
         converted: list | np.ndarray | None = None
         if dtype.kind == 'M':
-            if values.dtype.kind in 'if':# TODO: need to doublecheck if this is still needed
-                NATs = np.isnan(values)
-                converted = np.where(NATs, np.nan, values * 10e5).astype(dtype)
```

Comment:
So the NaT conversion isn't needed anymore?

Reply:
Indeed, the original code was as follows:

```python
                NATs = values == BOKEH_JS_NAT
                converted = np.where(NATs, np.nan, values * 10e5).astype(dtype)
```

Now Bokeh no longer uses a special value to mark NaT, it's just `np.nan`, so we no longer need to replace them with ... `np.nan` :)